app/domain/authentication/authn_jwt/jwt_configuration_interface.rb

```diff
@@ -2,9 +2,9 @@ module Authentication
   module AuthnJwt
     # Interface containing JWT configuration functions to be implemented in JWTConfiguration class for a vendor
     class JWTConfigurationInterface
```

Comment:
In ruby there is no interface, as it uses `duck-typing`.

Reply:
We used this us example 
https://github.com/conjurinc/dap-end2end-tests/blob/master/lib/end_to_end_tests/providers/provider_interface.rb
Its correct its not strict interface like java interface but it is a design idea we want to use